tensorrt_llm/_torch/auto_deploy/transform/library/export_to_onnx.py

```
@@ -212,25 +253,86 @@ def _translate_fake_quant_fp8_linear_op(
+        out = opset21.Add(out, bias)
+
+    return out
+
+
+def _translate_fake_quant_nvfp4_linear_op(
+    input: ir.Tensor,
+    weight_quantized: ir.Tensor,
+    bias: Optional[ir.Tensor],
+    input_scale: Sequence[ir.Tensor],
+    weight_scale: Sequence[ir.Tensor],
+    input_zp: Sequence[ir.Tensor],
```

Ruff (0.15.2)

[warning] Unused function argument: `input_zp`

(ARG001)